x-pack/plugins/observability/server/services/slo/create_slo.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import uuid from 'uuid';
+
+import { SLO } from '../../types/models';
+import { ResourceInstaller } from './resource_installer';
+import { SLORepository } from './slo_repository';
+import { TransformInstaller } from './transform_installer';
+
+import { CreateSLOParams, CreateSLOResponse } from '../../types/schema';
+
+export class CreateSLO {
```

Comment:
💬 Application service that orchestrates the use case of creating an SLO. Its role is to transform request payload into SLO model, calls the different services and transform the SLO model into the response model.

Reply:
https://github.com/elastic/kibana/pull/140523#discussion_r970779784